frontend/packages/dev-console/src/components/pipelines/detail-page-tabs/EventListenerDetails.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+import { SectionHeading, ResourceSummary } from '@console/internal/components/utils';
+import { referenceForModel } from '@console/internal/module/k8s';
+import { TriggerTemplateModel, TriggerBindingModel } from '../../../models';
+import { EventListenerKind } from '../resource-types';
+import ResourceLinkList from '../resource-overview/ResourceLinkList';
+
+export interface EventListenerDetailsProps {
+  obj: EventListenerKind;
+}
+
+const EventListenerDetails: React.FC<EventListenerDetailsProps> = ({ obj: eventListener }) => {
+  const { template, bindings } = (eventListener?.spec?.triggers || []).reduce(
+    (acc, trigger) => ({
+      bindings: [...acc.bindings, ...trigger.bindings.map((binding) => binding.name)],
```

Comment:
Oh right, my example neglected that it was an array. Hmmm... the array of objects is there because bindings can have two types ClusterTriggerBinding and TriggerBinding... can you add this to the tech debt ticket? I think we need to swing back once we get an operator update as ClusterTriggerBinding doesn't exist on 0.10.8 😞

Reply:
Task to update binding logic for 10.0.8 https://issues.redhat.com/browse/ODC-3557